### PR TITLE
add init submodule

### PR DESCRIPTION
**Pull request by vincentDcmps**
change recursive part to add init submodule

### CODE REVIEW

File: content/pipeline/docker/syntax/cloning/_index.md [hunk 1]
```diff
@@ -69,6 +69,7 @@ steps:
 - name: submodules
   image: alpine/git
   commands:
+  - git submodile init
```
Comment (tboerger) [inline content/pipeline/docker/syntax/cloning/_index.md:72]: There is a typo.

Comment (vincentDcmps) [reply content/pipeline/docker/syntax/cloning/_index.md]: done

**Comment by abrain** [comment]
Thanks for the PR @vincentDcmps ! I had to deal with this today and was about to submit the same.

The init command could also be included into the existing command, like so: `git submodule update --init --recursive`

**Comment by tboerger** [comment]
I think a single command would make more sense, like it have been mentioned in the previous comment.

**Review by tboerger — APPROVED** [review]

**Comment by tphoney** [comment]
thanks everyone for this 👍 , merging

**Pull request merged**